pom.xml

```diff
@@ -198,40 +198,23 @@
             </dependency>
 
             <!-- For sl4j/logback logging, see https://wiki.jasig.org/display/PLT/Logging+Best+Practices -->
-            <dependency>
-                <groupId>commons-logging</groupId>
-                <artifactId>commons-logging</artifactId>
-                <version>1.2</version>
-                <scope>provided</scope>
-            </dependency>
```

Comment:
For logging, the goal is logback (via the slf4j api). All other logging implementations (even transitive ones) should be removed. I'll look into how commons-logging is still getting deployed.

The JCL bridges (*-over-*) are now included in spring-jcl, so we don't have to explicitly declare them in the pom, however, I'll need to check if `jcl-over-slf4j-1.7.5.jar` is from spring-jcl, or a transitive dependency that should be excluded.

Reply:
So, lines 201-206 actually prevent any dependency from pulling in commons-logging, which is why I think it should remain.  Otherwise, you have to find which dependencies are pulling in commons-logging and add excludes.

Reply:
As per our side discussion with @bjagg , I'm going in the direction of using the maven enforcer plugin, and then excluding the commons-logging jars as needed. @groybal - please let me know if you don't feel this is sufficient.

Reply:
discussed in PR - change the parent pom to have a `dependencies` section with the excluded deps as 'provided'